codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithyjava/nameresolver/Dafny.java

```diff
@@ -304,7 +397,7 @@ TypeName typeForString(StringShape shape) {
         return typeForCharacterSequence();
     }
 
-    TypeName typeForCharacterSequence() {
+    public TypeName typeForCharacterSequence() {
```

Comment:
Note for Tony: I need to pull this down and grep around to see if this needs to be public.

Reply:
I pulled this PR down and found no public uses of this.
Lets leave this as private as possible.

```suggestion
   TypeName typeForCharacterSequence() {
```

Reply:
Good call, 'twas just leftover from earlier approach